### DISCUSSION TITLE
SlintHub for slint users and developers to share Components.

### DISCUSSION THREAD

**Discussion by evan0greenup** [slint-ui/slint#2948] [Ideas]
There is crates.io for Rust. For Slint, even if it can upload to crates.io, but It would be good to collect all the slint file together into SlintHub (or some name else). For users and developer to share Slint specific components with special tools.

**Comment by tronical** [comment]
I was thinking that instead of creating a new hub, Slint could support importing components from other git repositories. For example if we had some kind of import map I could say `@evanswidgets` is at `https://github.com/foo/bar.git` and then in my `.slint` file I could write `import { Blah } from "@evanswidgets/library.slint"`. The compiler would have to learn how to shallow fetch repos and cache them in a well-known location, but it would allow us to scale without a central service.

**Comment by ogoffart** [comment]
This also has been discussed there: https://github.com/slint-ui/slint/issues/154
